### ISSUE TITLE
When I turn on preload = true, it adds multiple preload lines?

### ISSUE THREAD

**Issue by isaackearl** [mitydigital/statamic-google-fonts#2]
<img width="980" alt="image" src="https://github.com/mitydigital/statamic-google-fonts/assets/9354830/47f8c255-7689-451c-a1b6-46f145a858cc">

package works great, but it adds extra lines, or maybe it's doing this and I don't realize that that is what it's supposed to do.  I also get this warning in the console:

The resource https://fonts.googleapis.com/css2?family=Cinzel&display=swap was preloaded using link preload but not used within a few seconds from the window's load event. Please make sure it has an appropriate `as` value and it is preloaded intentionally.
two times.

Thanks for the package!  is this how it is supposed to work?

**Comment by martyf** [comment]
The purpose of this addon is to _not_ have direct links to Google. The Spatie package handles all of this, and this package is simply a wrapper to bring it across to Statamic.

In the current site I'm working on, within my `<head>`, I have:
```antlers
<head>
    <!-- metadata, title, etc -->

    {{ googlefonts }}

    <!-- anything else you may have - tailwind, js, etc -->
</head>
```

This outputs a <style> with fonts being loaded directly from the site, such as:
```html
<style>/* cyrillic-ext */
@font-face {
  font-family: 'Inter';
  font-style: normal;
  font-weight: 400;
  font-display: swap;
  src: url(http://my-site.test/storage/fonts/9363b656ec/sinterv13ucc73fwrk3iltehus-fvqtmwcp50knma2jl7w0q5n-wu.woff2) format('woff2');
  unicode-range: U+0460-052F, U+1C80-1C88, U+20B4, U+2DE0-2DFF, U+A640-A69F, U+FE2E-FE2F;
}
<!-- ... continues ... -->
```

Not sure what you're doing to get all of that output though. 

Have you looked at the four points at the bottom of the installation docs:
https://docs.mity.com.au/google-fonts/installation

And then then you just need to implement with the `{{ googlefonts }}` tag - docs are at:
https://docs.mity.com.au/google-fonts/tag

**Comment by isaackearl** [comment]
hey @martyf ok thanks for the info.  Yes the package works exactly as you describe when I have the prefetch option turned off.  I used this same package in a regular laravel project about a month ago, so your comment prompted me to go and check that out and see if I was having the same issue there.  I discovered that 1 month ago that prefetch option did not exist, so that is brand new in the underlying spatie configuration.  

I updated the other app, and deleted my google-fonts.php file and re-published it, and it added the new 'preload' option in there.  When I tried to use it does the same exact thing.  In other words, this bug is part of the underlying package and I should go and open my issue there instead of here.  

Thanks!
